lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -78,6 +79,25 @@ def test_as_honor_for_staff(self):
         self.setup_user(self.honor_user)
         self.verify_response(expected_status_code=403, params={'username': self.staff_user.username})
 
+    def test_filter_by_org(self):
+        """Verify that CourseDescriptors are filtered by the provided org key."""
+        self.setup_user(self.staff_user)
+
+        # Create a second course to be filtered out of queries.
+        alternate_course = CourseFactory.create()
+
+        # No filtering.
+        unfiltered_response = self.verify_response()
+        self.assertTrue(
+            any(course['org'] == alternate_course.org for course in unfiltered_response.data['results'])
+        )
+
+        # With filtering.
+        filtered_response = self.verify_response(params={'org': self.course.org})
+        self.assertTrue(
+            all(course['org'] == self.course.org for course in filtered_response.data['results'])
+        )
```

Comment:
Do `alternate_course` and `self.course` have different values for org?
Since both of your tests are positive tests, where do you make sure that not-wanted orgs are actually filtered out?